xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/test/TestConversionMatrix.cpp

```diff
@@ -128,21 +143,21 @@ TEST(TestConvertMatrix, YUV2RGB)
   yuvMat = convMat.GetYuvMat();
   DebugPrint(yuvMat);
 
-  EXPECT_EQ(bt709Mat_8bit, yuvMat);
+  EXPECT_TRUE(CompareMatrices(bt709Mat_8bit, yuvMat, 0.01f));
```

Comment:
Maybe is safe use more small number i.e. `0.0001f`  (rounding errors for type of representation are much smaller than this)

Reply:
I guess I will leave this part with intact `FloatEquals` and `0.0001f` as threshold and instead of hacking `FloatEquals` I will make use of `std::round` in EDL functiins to make the output predictable.